src/averageAges.js

```diff
@@ -20,6 +20,24 @@ function calculateMenAverageAge(people, century) {
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+  // const men = people.filter((a) => a.sex === 'm');
+  const men = century
+    ? people.filter((man) => {
+      const centuryDied = Math.ceil(man.died / 100);
+
+      return man.sex === 'm' && centuryDied === century;
+    })
+    : people.filter((a) => {
+      return a.sex === 'm';
+    });
+
+  const sum = men.reduce((count, man) => {
```

Comment:
in my opinion, you use too many variables

Reply:
Will be better to specify that it's sum of ages)